message_adapter/message_adapter.py

```diff
@@ -424,7 +429,8 @@ def createNextEvent(self, handlerResponse, event, messageConfig):
         """
         self.__validate_json(handlerResponse, 'output')
         result = self.__assignOutputs(handlerResponse, event, messageConfig)
-        result['exception'] = 'None'
+        if ('exception' not in result):
+            result['exception'] = 'None'
```

Comment:
So we are storing the exception to the remote message now.

Reply:
It is already being stored in the short message in certain cases. This change is to make sure it preserves the exception when loading the full message from S3.